examples/Atlas/+atlasControllers/@InstantaneousQPController/velocityReference.m

```diff
@@ -0,0 +1,52 @@
+function v_ref = velocityReference(obj, t, q, qd, qdd, foot_contact_sensor, params)
+  % Integrate expected accelerations to compute a feed-forward velocity reference. 
+  % @param t time (s)
+  % @param q, qd robot state
+  % @param qdd accelerations computed by the QP
+  % @param foot_contact_sensor 2x1 flags indicating whether contact is detected
+  %                            for the [left; right] foot.
+  % @param params the vref_integrator field of an atlasParams.* object.
+  fc = struct('right', foot_contact_sensor(2) > 0.5,...
+                        'left', foot_contact_sensor(1) > 0.5);
+
+  if isnan(obj.vref_integrator_data.t_prev)
+    obj.vref_integrator_data.t_prev = t;
+  end
+  dt = t - obj.vref_integrator_data.t_prev;
+  obj.vref_integrator_data.t_prev = t;
+
+  qd_int = obj.vref_integrator_data.state;
+  qd_int = (1-params.eta)*qd_int + params.eta*qd + qdd*dt;
+
```

Comment:
Note: having params.eta and params.integrator.eta in wholeBodyPID could be a bug source later on (I know you were going off the variable names I chose originally).